main.css

```diff
@@ -0,0 +1,746 @@
+:root {
+  --main-font-family: 'Lato', sans-serif;
+  --heading-font-family: 'Cocogoose', sans-serif;
+  --orange-shade-color: #ec5242;
+  --blue-shade-color: #272a31;
+  --gray-shade-color: #d3d3d3;
+  --white-color: #fff;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  font-family: var(--main-font-family);
+  color: #333;
+}
+
+.wrapper {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+header {
+  width: 100%;
+  margin: 1em 0;
+  height: 63px;
+}
+
+.mobile-nav {
+  height: 100%;
+  width: 100%;
+}
+
+.desktop-nav {
+  display: none;
+}
+
+main {
+  background: url('./media/backgrounds/mobileBackground1\ \(3\).png') no-repeat top;
+  position: absolute;
+  top: 80px;
+  height: auto;
+}
+
+.partners-section,
+footer {
+  display: none;
+}
+
+.with-hamburger {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.mobile-logo {
+  width: 150px;
+  height: 51px;
+  margin-right: 10px;
+}
+
+.mobile-logo-img {
+  width: 100%;
+  height: 100%;
+}
+
+.menu-button {
+  width: 40px;
+  height: 40px;
+  color: var(--blue-shade-color);
+  display: flex;
+  border: none;
+  background: none;
+  cursor: pointer;
+  z-index: 9;
+  margin-left: 20px;
+}
+
+.ham {
+  width: 40px;
+  height: 40px;
+  transition: 0.3s ease-in-out;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.bar1,
+.bar2,
+.bar3 {
+  width: 100%;
+  height: 3px;
+  margin: 5px 0 3px 0;
+  background: var(--blue-shade-color);
+  border-radius: 6px;
+}
+
+.action-slide .bar1 {
+  transform: rotate(-45deg) translate(-3px, 14px);
+  background: var(--orange-shade-color);
+}
+
+.action-slide .bar3 {
+  transform: rotate(45deg) translate(0, -12px);
+  background: var(--orange-shade-color);
+}
+
+.action-slide .bar2 {
+  opacity: 0;
+}
+
+.olist {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 42px;
+  font-size: 50px;
+  padding: 30px 20px 20px 20px;
+}
+
+.olist-a {
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.olist-a:link {
+  color: var(--blue-shade-color);
+}
+
+.olist-a:visited {
+  color: var(--orange-shade-color);
+}
+
+.olist-a:active {
+  color: #000;
+}
+
+.slidediv {
+  width: 100%;
+  height: 100vh;
+  background: #fff;
+  color: var(--blue-shade-color);
+  z-index: 2;
+  position: absolute;
+  top: 0;
+  transform: translateX(-110%);
+  transition: 0.2s;
+  padding: 60px;
+}
+
+.active-slide {
+  transform: translateX(0);
+}
+
+.head-section {
+  text-align: center;
+  padding-top: 51px;
+  width: 90%;
+  margin: 0 auto;
+}
+
+.headline-h2 {
+  color: var(--orange-shade-color);
+}
+
+.headline-h1 {
+  font-size: 2em;
+  background-image: url('./media/backgrounds/imagefortext.jpg');
+  background-size: cover;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
```

Comment:
- [ ] On the main page, these main titles should be aligned to the left. Also, your h1 title should be written in the uppercase. Kindly fix it to match the given design: 
yours: 
![image](https://github.com/tan12082001/Project-Capstone-1/assets/91486791/c31e42d5-d56a-4ce7-8fae-4f0fdcc5edf3)
given:
![image](https://github.com/tan12082001/Project-Capstone-1/assets/91486791/5799f87f-c2b3-4289-ab6c-d5dccdbb1e51)